### DISCUSSION TITLE
Pre processing of OSM data

### DISCUSSION THREAD

**Discussion by map-per** [organicmaps/organicmaps#5979] [Q&A] [answered]
Do you do some pre processing (in addition to replaced_tags.txt) of OSM data for the production map data? Like e.g. converting contact:housenumber to addr:housenumber or adding area=yes to multipolygon pedestrian areas?

**Comment by map-per** [comment]
I'm asking because when I use the latest release version (2023.08.18-8-android) for map generation multipolygon `pedestrian` areas without `area=yes` don't show up:
![Bildschirmfoto vom 2023-09-16 12-41-26](https://github.com/organicmaps/organicmaps/assets/79519062/6606c755-2277-426a-b52a-b63a8ecbf02c)
(https://www.openstreetmap.org/relation/1951581#map=18/48.13418/11.56733)

**Comment by vng** [comment, marked as answer]
We have a separate branch on the generator server with a bunch of experimental changes. So the generator output from the master branch may be a bit different. We usually put tested changes from generator into master in some time.

**Reply by map-per** [reply]
Ok, that explains a lot... some OM behaviour really confused me. Can you push that branch? That would help me test #5884.
